frontend/src/layouts/BaseLayout.vue

```diff
@@ -237,7 +256,8 @@ export default defineComponent({
   name: 'BaseLayout',
   components: { AddressSettings, ArgentWarningModal, BaseButton, BaseTooltip, ConnectWallet, HeaderLinks, NetworkDropdown }, // prettier-ignore
   setup() {
-    const { advancedMode, isDark, toggleAdvancedMode, toggleDarkMode } = useSettingsStore();
+    const test = 'test'
```

Comment:
What's this?

Reply:
Removed the unused variable here: https://github.com/ScopeLift/umbra-protocol/pull/331/commits/84f5fc3f8818f9cbb64240ca4a79d4dd3faccdfe